src/TableRow.js

```diff
@@ -0,0 +1,31 @@
+import React, { useState } from "react";
+import moment from "moment";
+
+const Row = ({ booking }) => {
```

Comment:
Really great job! Can you explain why extracting the table row to a component helps solve the exercise?

Reply:
Otherwise each row would have only one state and selecting by each would not be possible. Now , each row will be its own state to be updated / to be used for any reason. We can then separate the rows easily .